cdap-docs/developers-manual/source/building-blocks/index.rst

```diff
@@ -55,10 +55,10 @@ write data through the data abstraction layer in CDAP.
 An :doc:`Artifact <artifacts>` is a jar file that packages the Java Application class, as well
 as any other classes and libraries needed to create and run an Application. 
 
-:doc:`Metadata <metadata>` |---| consisting of **properties** (a list of key-value pairs)
+:doc:`Metadata <metadata-lineage>` |---| consisting of **properties** (a list of key-value pairs)
 or **tags** (a list of keys) |---| can be set for datasets, streams, and applications.
 These can be retrieved and searched, and the metadata used to discover CDAP entities.
-Access of these entities is tracked, and you can view the lineage of datasets and streams.
+Access of these entities is tracked, and you can view the :doc:`lineage <metadata-lineage>` of datasets and streams.
 With a lineage, you can then drill down into the metadata of its nodes. 
```

Comment:
yeah, I mentioned REST API response though, primarily because we do not have lineage diagrams on the UI yet.